docs/api/c/data_chunk.md

```diff
@@ -4,7 +4,7 @@ title: C API - Data Chunks
 selected: Data Chunks
 ---
 
-Data chunks represent a horizontal slice of a table. They hold a number of vectors, that can each hold up to the `VECTOR_SIZE` rows. The vector size can be obtained through the `duckdb_vector_size` function and is configurable, but is usually set to `1024`.
+Data chunks represent a horizontal slice of a table. They hold a number of vectors, that can each hold up to the `VECTOR_SIZE` rows. The vector size can be obtained through the `duckdb_vector_size` function and is configurable, but is usually set to `2048`.
```

Comment:
Seems like the VECTOR_SIZE was bumped to 2048 recently?
See https://github.com/duckdb/duckdb/commit/e6b8a950c4b1c265def808ba6262c20ac21c38b1